tf/enclave/variables.tf

```diff
@@ -21,7 +21,7 @@ variable resource_group_name {
 }
 
 variable location {
-  default = "Central US"
+  default = "South Central US"
```

Comment:
South Central US has an access feature I want to try out.  It's in San Antonio(ish) and fairly centralized on Azure's backbone.